mesh.proto

```diff
@@ -638,9 +638,10 @@ message Waypoint {
   uint32 expire = 4;
 
   /*
-   * If true, only allow the original sender to update the waypoint.
+   * If greater than zero, treat the value as a nodenum only allowing them to update the waypoint.
+   * If zero, the waypoint is open to be edited by any member of the mesh.
    */
-  bool locked = 5;
+  fixed32 locked_to = 5;
```

Comment:
Isn't node number a UInt32?

Reply:
In some areas, and others not. This brings up a point though. Fixed32 is more space efficient for larger values. Most folks won't lock probably, so we'll do uint32.